modules/determining-upgrade-viability-cv-conditiontype.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="understanding-clusterversion-conditiontypes_{context}"]
+= Understanding cluster version condition types
+ 
+The Cluster Version Operator (CVO) is responsible for collecting the status conditions of the cluster version and its Operators. This status includes the condition type, informing you of the health and current state of the {product-title} cluster.
+
+In addition to `Available`, `Progressing`, and `Upgradeable`, there are additional condition types that affect cluster versions and Operators.
+
+* Failing: 
+The CVO monitors cluster Operators and other components. `Failing` is set on the cluster version status when a cluster cannot reach the desired state. This indicates that the cluster is not healthy, and requires an administrator to intervene.
+
+* Invalid:
+If the the cluster version has an error that prevents the server from taking action, the condition type of the cluster version reports as `ClusterVersionInvalid`. The CVO only reconciles the current state as long as this condition is set.
```

Comment:
the the -> the
The condition type might be named Invalid. @evakhoni for double check.

Reply:
updated :)